runtime/queries/groovy/locals.scm

```diff
@@ -0,0 +1,6 @@
+(function_definition) @local.scope
+
+(parameter
+  name: (identifier) @local.definition.parameter)
```

Comment:
```suggestion
  name: (identifier) @local.definition)
```

we only support `local.scope`, `local.definition` and `local.reference` in locals queries